SR2020/2020-06-18-version-0-2-0-simulator.md

```diff
@@ -0,0 +1,23 @@
+---
+to: sr2020-teams
+subject: Virtual Competition Simulation Version 2 Released
+---
+
+Hi
+
+We have released version 0.2.0 of the competition simulation which you can download from the [Docs page](https://studentrobotics.org/docs/competition-simulator/).
+
+The following changes have been made:
```

Comment:
There are technically some other changes they may notice, but I think these are the ones they really care about

```suggestion
We've made a number of changes, most notably:
```